app/Http/Controllers/PostsController.php

```diff
@@ -85,6 +85,21 @@ public function index(Request $request)
             $query = $query->withTag($filters['tag']);
         }
 
+        // If there is a limit set on siblings, include in query.
+        if ($request->query('include')) {
+            foreach ($request->query('include') as $key => $value) {
+                if ((strpos($value, 'siblings') !== false) && (strpos($value, 'limit') !== false)) {
+
+                    $limit = (new \League\Fractal\Manager)
+                            ->parseIncludes($value)
+                            ->getIncludeParams('siblings');
+
+                    $limit = $limit ? (int) $limit->get('limit')[0] : null;
+
+                    $query = $query->withPostLimit($limit);
```

Comment:
@DFurnes working on [setting a limit when grabbing a post's siblings](https://www.pivotaltracker.com/n/projects/2019429/stories/159406186) so this doesn't break the inbox! 

I'm having trouble getting this to work and wanted a quick gut check to get your thoughts on doing it this way (closely following how we did the work here to parse by post types [here](https://github.com/DoSomething/rogue/pull/752)). 

Is this a reasonable way to do this?

Reply:
TBH, I'm not sure this needs to be configurable per request. If you want, just adding a `->take(10);` to the [siblings relationship](https://github.com/DoSomething/rogue/blob/6455f3c742b68c89703d1bb740135ec6f8ce14eb/app/Models/Post.php#L173-L179) should do the trick without too much added complexity!

Reply:
ah that makes total sense I was definitely overthinking! Updated and also the documentation! :)